linux/create_installer_linux.sh

```diff
@@ -0,0 +1,135 @@
+#! /bin/bash
```

Comment:
Has this whole file had shellcheck run on it?

Reply:
Yes, but I missed one issue. It's fixed now.